cupy/_sorting/search.py

```diff
@@ -3,6 +3,7 @@
 from cupy.core import fusion
 from cupy import util
 
+from  cupy.core._routines_indexing import _ndarray_argwhere
```

Comment:
Small style fix from flake8 on Travis CI

```suggestion
from cupy.core._routines_indexing import _ndarray_argwhere
```

Reply:
Oh yes. But on testing and in `continuous-integration/travis-ci/pr` also there is an `ImportError` for this? I am trying to find out the mistake.